### PR TITLE
Properly apply Apache-2.0 license

### LINKED ISSUE #227: Fixing the ugly badges
Namely, these two:

* License Badge
* Doc Badge

The first one was due to our forgetting filling the license template for ASL 2.0, resulting in the license being reverted to a placeholder ... and GitHub failing to recognize that. Can be easily fixed.

The second, though ... not sure why docs are marked as "failing".

Maybe because the documentation has been updated by #223 but has not been pushed to readthedocs?

### PR DESCRIPTION
**Pull request by pepoluan**
## What do these changes do?

Properly implement Apache-2.0 license according to [its guidelines (for non-Apache software)](http://www.apache.org/foundation/license-faq.html#Apply-My-Software).

## Are there changes in behavior for the user?

Should be none unless user actually relies on the (e.g.) hash of the `LICENSE` file (why???)

## Related issue number

Closes #227 

## Checklist

None is applicable.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,13 +1,176 @@
...
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
```
Comment (pepoluan) [inline LICENSE:176]: Remove the whole appendix containing the boilerplate code.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,5 @@
+aiosmtpd
+Copyright 2014-2021 The aiosmtpd Developers
```
Comment (pepoluan) [inline NOTICE:2]: This provides the necessary details for the "Licensor" in `LICENSE`

**Review by pepoluan — COMMENTED** [review]
I've provided comments on some points that resulted in earlier "pulling out" of the `LICENSE` file.

I believe the salient issues have been fixed, and thus we have fulfilled the legal obligations to legally apply the Apache-2.0 license.

**Review by waynew — APPROVED** [review]
📜 🚀

**Pull request merged**